e2e/reset-modal.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { test, expect } from '@playwright/test';
+import translations from '../client/i18n/locales/english/translations.json';
+
+const resetText = translations.learn['reset'];
+const resetWarnText = translations.learn['reset-warn'];
+const resetWarnText2 = translations.learn['reset-warn-2'];
+const resetLessonButtonText = translations.buttons['reset-lesson'];
+
+async function seed() {
+  // ...
+}
+
+test.beforeAll(async () => {
+  await seed();
+});
+
+test.beforeEach(async ({ page }) => {
+  await page.goto(
+    '/learn/2022/responsive-web-design/learn-html-by-building-a-cat-photo-app/step-1'
```

Comment:
```suggestion
    '/learn/2022/responsive-web-design/learn-html-by-building-a-cat-photo-app/step-2'
```

The first step has the project preview modal, which might be what's interfering with the tests here.

Reply:
> The first step has the project preview modal, which might be what's interfering with the tests here.

its not working, showing some out of viewport in safari and chromium

Reply:
Have you tried updating the step URL as I've indicated above?

Reply:
Yes, but tests are failing in webkit and safari